src/emulator/config/include/config/config.h

```diff
@@ -43,15 +40,24 @@ struct Config {
     bool pstv_mode = false;
     bool show_gui = false;
     bool show_game_background = true;
-    std::string pref_path = {};
-    fs::path config_path = {};
-    bool overwrite_config = true;
-    bool load_config = false;
+    int icon_size = 64;
     bool archive_log = false;
     bool texture_cache = true;
-    optional<bool> wait_for_debugger;
+    int sys_button = 1;
+    int sys_lang = 1;
     std::string background_image = {};
     float background_alpha = 0.300f;
-    int icon_size = 64;
+    int log_level = 0;
```

Comment:
why reorder all this ?

Reply:
Consistency. In `config.cpp` it is ordered as it is now in `config.h`.

Reply:
ah, i see